modules/realsense_d435.py

```diff
@@ -0,0 +1,192 @@
+import pyrealsense2 as rs
+import traceback
+import sys
+import time
+import numpy as np
+
+class rs_d435:
```

Comment:
Is there anywhere you can write down the maths for this? Where the formulas come from, general coordinate space etc?

Reply:
I guess I should put a markdown document together about both cameras. It's is a little complicated to explain though. This is where the [deprojection](https://github.com/IntelRealSense/librealsense/blob/7148f9ae1d78b5d44bee4fc578bf0b8fb9a220c5/wrappers/python/examples/box_dimensioner_multicam/helper_functions.py#L121) comes from

Reply:
First stab at documentation is here 4cad3967ded27b7fe7eef6c871b2c15e4721b04c

Reply:
Happy with that